PHP/PHP Source.sublime-syntax

```diff
@@ -2576,12 +2576,14 @@ contexts:
         2: meta.regex.modifier.php
         3: punctuation.definition.string.end.php
       pop: 1
-    - match: \"
+    - match: /|"
       fail: branch-point-double-quoted
     - match: ''
       push: scope:source.regexp.php
       with_prototype:
-        - match: (?=/{{regex_modifier}}"|")
+        # for syntax test in regex
+        - match: // *\^.*$
```

Comment:
This is to deal with https://github.com/sublimehq/Packages/blob/41b7232d08850f9387c86b622dcdc949c188498d/PHP/tests/syntax_test_php.php#L5127-L5135 so that the test can continue to work without being failed due to unescaped `/` in a regex.